Extensions/CursorType.json

```diff
@@ -36,26 +36,125 @@
           "disabled": false,
           "folded": false,
           "type": "BuiltinCommonInstructions::JsCode",
-          "inlineCode": "var styleCursor = eventsFunctionContext.getArgument(\"style_cursor\") || \"default\" ;\nif (typeof document !== \"undefined\") {\n    document.body.style.cursor = styleCursor;\n}",
+          "inlineCode": "var styleCursor = eventsFunctionContext.getArgument(\"style_cursor\") || \"default\";\nif (typeof document !== \"undefined\") {\n    document.body.style.cursor = styleCursor;\n}",
           "parameterObjects": "",
-          "useStrict": true,
-          "eventsSheetExpanded": false
+          "useStrict": true
         }
       ],
       "parameters": [
         {
           "codeOnly": false,
-          "defaultValue": "",
-          "description": "The name of the new cursor (\"default\", \"pointer\", \"help\", etc...)",
-          "longDescription": "",
+          "defaultValue": "default",
+          "description": "The new cursor type",
+          "longDescription": "List of available cursors on https://developer.mozilla.org/en-US/docs/Web/CSS/cursor",
           "name": "style_cursor",
           "optional": false,
-          "supplementaryInformation": "",
-          "type": "string"
+          "supplementaryInformation": "[\"auto\", \"default\", \"none\", \"context-menu\", \"help\", \"pointer\", \"progress\", \"wait\", \"cell\", \"crosshair\", \"text\", \"vertical-text\", \"alias\", \"copy\", \"move\", \"no-drop\", \"not-allowed\", \"grab\", \"grabbing\", \"all-scroll\", \"col-resize\", \"row-resize\", \"n-resize\", \"e-resize\", \"s-resize\", \"w-resize\", \"ne-resize\", \"nw-resize\", \"se-resize\", \"sw-resize\", \"ew-resize\", \"ns-resize\", \"nesw-resize\", \"nwse-resize\", \"zoom-in\", \"zoom-out\"]",
+          "type": "stringWithSelector"
         }
       ],
       "objectGroups": []
     }
   ],
-  "eventsBasedBehaviors": []
+  "eventsBasedBehaviors": [
+    {
+      "description": "Sets the cursor appearence on hover of the object.",
+      "fullName": "Hover cursor",
+      "name": "CursorHover",
+      "objectType": "",
+      "eventsFunctions": [
+        {
+          "description": "",
+          "fullName": "",
+          "functionType": "Action",
+          "name": "doStepPreEvents",
+          "sentence": "",
+          "events": [
+            {
+              "disabled": false,
+              "folded": false,
+              "type": "BuiltinCommonInstructions::JsCode",
+              "inlineCode": "objects[0]\n    .getRendererObject()\n    .cursor = objects[0]\n        .getBehavior(\n            eventsFunctionContext.getBehaviorName(\"Behavior\")\n        )\n        ._getcursor_type();\n",
```

Comment:
It's ok to use `_getcursor_type`  here because the underscore is here to avoid external code, i.e: JS code of the scenes, to get access to the properties (if you want to expose a property, you can make an expression for it).
Here, we're inside the behavior, so we're allowed to access this "private" property.